check_monotonicity/CheckMonotonicityModule.py

```diff
@@ -0,0 +1,56 @@
+from torch import nn
+import torch.nn.functional as F
+
+
+class CheckMonotonicityModule(nn.Module):
+    def __init__(
+        self,
+        layer, 
+        layer_idx, 
+        lm_head, 
+        final_layer_norm,
+        plot_monotonicity
+    ):
+        super().__init__()
+
+        self.layer = layer
+        self.layer_idx = layer_idx
+        self.lm_head = lm_head
+        self.final_layer_norm = final_layer_norm
+        self.plot_monotonicity = plot_monotonicity
+        self.sentence = None
```

Comment:
I would be really grateful if you'd add a docstring describing what arguments are expected (a few words would be enough). Although most of the make sense, foe example I'm not sure what `plot_monotonicity` is supposed to be. It looks like a boolean but later you use it as follows: `self.plot_monotonicity.save_most_likely_tokens(most_likely_tokens)`. So I assume it is some sort of object.